README.md

```diff
@@ -21,7 +21,7 @@ If you are running into an issue and cannot find any information in the troubles
 
 ## Cromwell on Azure 
 
-[Cromwell](https://cromwell.readthedocs.io/en/stable/) is a workflow management system for scientific workflows, orchestrating the computing tasks needed for genomics analysis. Originally developed by the [Broad Institute](https://github.com/broadinstitute/cromwell), Cromwell is also used in the GATK Best Practices genome analysis pipeline. Cromwell supports running scripts at various scales, including your local machine, a local computing cluster, and on the cloud. <br/>
+[Cromwell](https://cromwell.readthedocs.io/en/stable/) is a workflow management system for scientific workflows, orchestrating the computing tasks needed for genomics analysis. Originally developed by the [Broad Institute](https://github.com/broadinstitute/cromwell), Cromwell is also used in the GATK Best Practices genome analysis pipeline. Cromwell supports running scripts at various scales, including your local machine, a local computing cluster, and on the cloud. Cromwell workflows are written using the open source [WDL](https://github.com/openwdl/wdl) or the [CWL](https://www.commonwl.org/) scripting languages. To see examples of WDL scripts - see this ['Learn WDL'](https://github.com/openwdl/learn-wdl) Repo on Github<br/>

```

Comment:
Hello, thank you for this.  Could you please move your changes to the end of the next paragraph, since it already mentions WDL there.  Also, could you please fix the following minor spelling/grammar issues:

- "Repo" => repository
- "Github" => "GitHub"
- Add a period to the end of the last sentence